docs/cvl/expr.md

```diff
@@ -359,3 +359,11 @@ state `s`.
 Unresolved method calls
 ```
 
+(storage-comparison)=
+Comparing storage
+-----------------
+
+```{todo}
+Merge #78
```

Comment:
I don't think we want that to reach public docs

Reply:
This is docs PR #78 - should be replaced with a link to the update documentation (which has landed)